usaspending_api/conftest_helpers.py

```diff
@@ -63,12 +64,12 @@ def _add_contents(self):
     @staticmethod
     def _refresh_materialized_views():
         """
-        This materialized view is used by transaction_delta_view.sql, so
-        we will need to refresh it in order for transaction_delta_view to see
+        This materialized view is used by the es etl view, so
+        we will need to refresh it in order for the view to see
```

Comment:
Any special reason we wanted to be more generic here?

Reply:
A few hardcoded vars were consolidated and stored in settings.py. Code comments can easily become outdated from hardcoded values as well.